docs/Makefile

```diff
@@ -21,19 +13,5 @@ build:
 preview:
 	cd ../ && mkdocs serve
 
-%.png: %.pdf
-	convert -density 300 $< $@
-
-%.pdf: %.gv
-	dot -Tpdf $< > $@
-
 sphinx:
```

Comment:
what is this for?

Reply:
This is for generating https://github.com/PrairieLearn/PrairieLearn/blob/master/docs/python-grader/sphinx-docs.md. I think it'd be fine to try to make mkdocs do this automatically, though it's a little nice to be able to review the generated documentation during code review. That said, we currently have nothing in CI enforcing that it stays up to date, so there's no guarantee that it's actually kept up to date with the underlying source 🙃